controllers/clusterversion_controller.go

```diff
@@ -124,47 +122,181 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/reconcile
 func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	var err error
 	c.ctx = ctx
 	c.log = log.FromContext(ctx, "ClusterVersion", req)
-	c.log.Info("Reconciling ClusterVersion")
 
-	if err := c.ensureConsolePlugin(); err != nil {
-		c.log.Error(err, "unable to deploy client console")
+	c.log.Info("Loading input resources")
+
+	// Load the cluster version
+	c.clusterVersion.Name = req.Name
+	c.clusterVersion.Namespace = req.Namespace
```

Comment:
clusterversion is clusterscoped, no need for namespace.

Reply:
done